ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,30 @@
+name: 🐛 Bug
+description: Report an issue to help improve the project.
+labels: ["🛠 goal: fix"]
```

Comment:
These labels do not exist on this repository. 👀
```suggestion
labels: ["bug"]
```

Reply:
Okay.

Reply:
@Panquesito7 these are fallback files for other repos 🙂